firefox-ios/Client/Frontend/Home/Homepage Rebuild/TopSites/TopSiteCell.swift

```diff
@@ -0,0 +1,280 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import Shared
+import SiteImageView
+import Storage
+import UIKit
+
+/// The TopSite cell that appears for the homepage rebuild project.
+class TopSiteCell: UICollectionViewCell, ReusableCell {
```

Comment:
this file is a copy of `TopSiteItemCell`, but changed the name and update the `configure` method to using `TopSiteState` instead of `TopSite`